guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md

```diff
@@ -298,32 +294,32 @@ class LoginCustomerOnFrontendStep implements TestStepInterface
  
 ### `constructor()` method  {#constructor-method}
 
-You can optionally use the `constructor()` method which injects data to be used by `run()`. The data may include classes like pages, variation data, data returned by previous executed steps, etc.
+You may optionally use the `constructor()` method which injects data to be used by `run()`. The data may include classes like pages, variation data, data returned by previous executed steps, etc.
```

Comment:
splitting?

"Optionally, you may use..."

Reply:
comma before "which"

Tells us what `run()` is like the constructor. Maybe "...the `run()` method."

Reply:
^^